benchmark/Streamly/Benchmark/Data/Unbox.hs

```diff
@@ -0,0 +1,102 @@
+{-# LANGUAGE TemplateHaskell #-}
+
+-- |
+-- Module      : Streamly.Benchmark.Data.Unbox
+-- Copyright   : (c) 2023 Composewell
+-- License     : MIT
```

Comment:
Chnage the license to BSD-3. Can you also change this in any other benchmark modules? Other this will keep propagating by copy/apste.

Reply:
See https://github.com/composewell/streamly/issues/2481